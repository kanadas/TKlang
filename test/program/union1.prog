f :: int -> int -> char + int + bool;
f a b = if a + b == 0 then 1 @ 'a' else 2 @ a + b

