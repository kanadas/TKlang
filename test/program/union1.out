program/union1.prog
No main stream. Nothing to do...
